app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -77,3 +77,4 @@ alwaysAllow=Always allow
 alwaysDeny=Always deny
 appearanceSettings=Appearance settings
 autoHideMenuBar=Hide the menu bar by default
+disableTitleMode=Disable Title Mode
```

Comment:
I'm not sure people will know what 'Title Mode' is. Maybe change the text to 'Always show URL bar'?